src/zen/tlsmanager.h

```diff
@@ -20,16 +20,16 @@ using namespace std;
 
 namespace zen
 {
-typedef struct _NODE_ADDR {
-    std::string ipAddr;
-    int64_t time; // time in msec, of an attempt to connect via TLS
+// typedef struct _NODE_ADDR {
```

Comment:
The `struct _NODE_ADDR` seems to me has been completely moved to `net.h`, let's remove the commented code definitely.

Reply:
yes, I forgot to remove that, thanks.